src/utils/browserslist.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Converts esbuild target [^1] (which is also used by Rolldown [^2]) to browserslist target [^3].
+ *
+ * [^1]: https://esbuild.github.io/api/#target
+ * [^2]: https://github.com/rolldown/rolldown/blob/v1.0.0-beta.8/packages/rolldown/src/binding.d.ts#L1429-L1431
+ * [^3]: https://github.com/browserslist/browserslist
+ */
+export function esbuildTargetToBrowserslist(
+  target: string | string[],
```

Comment:
```suggestion
  target: string[],
```
Always be `string[]`